### PR TITLE
feat(CI): Add UI to Circle CI (AEROGEAR-8370)

### PR DESCRIPTION
**Pull request by damienomurchu**
## Motivation
UI missing from CI config (AEROGEAR-8370)

## What
Add CI build checks for react UI

## Why
Keep those frontend devs honest

## How
Update CircleCI config and Makefile

## Verification Steps
Check CircleCI build details

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task

### CODE REVIEW

File: ui/README.adoc [hunk 1]
```diff
@@ -8,9 +8,21 @@ ifdef::env-github[]
...
 
+:toc:
+:toc-placement!:
 
 = Mobile Security Service UI
 
+ifdef::status[]
+.*Project health*
+image:https://circleci.com/gh/aerogear/mobile-security-service.svg?style=svg[Build Status (CircleCI), link=https://circleci.com/gh/aerogear/mobile-security-service]
+image:https://coveralls.io/repos/github/aerogear/mobile-security-service/badge.svg?branch=master[Coverage Status (Coveralls), link=https://coveralls.io/github/aerogear/mobile-security-service?branch=master]
```
Comment (davidffrench) [inline ui/README.adoc]: I am not sure if coveralls is only reporting Go test coverage or not. Do you know the answer?

Comment (damienomurchu) [reply ui/README.adoc]: good question @davidffrench, its only go right now, but 64% on badge is inconsistent with 96% reported coverage on [coveralls](https://coveralls.io/github/aerogear/mobile-security-service?branch=master). Will investigate

Comment (damienomurchu) [reply ui/README.adoc]: @davidffrench best to remove this right now (the code cov badge) as its not related to the UI I think, so will do so now

Comment (davidffrench) [reply ui/README.adoc]: Agreed

**Pull request merged**